metadata/blinky.ukp.informatik.tu-darmstadt.de%2Fshibboleth.xml

```diff
@@ -7,6 +7,9 @@
             <saml:Attribute xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion" Name="http://macedir.org/entity-category" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:uri">
```

Comment:
Perhaps the cleanest is to declare `xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion"` on the parent element  `<EntityAttributes>` or in the root element `<md:EntityDescriptor>` then it will be available for all `<saml:Attribute>` elements from where it can be removed from here.

Reply:
I just copied the one from weblicht and added the namespaces to the root as you suggested.

Reply:
Sorry. Indeed my linked example was not the most compatible in terms of namespace declarations. I missed that before.
It all looks good now.